class NetworkError(Exception):
  pass

class CommandError(Exception):
  pass

class UsageError(Exception):
  pass